canonicalwebteam/store_api/store.py

```diff
@@ -57,15 +57,20 @@ def get_endpoint_url(self, endpoint, api_version=1):
         base_url = self.config[api_version]["base_url"]
         return f"{base_url}{endpoint}"
 
-    def find(self, query="", category="", fields=[]):
+    def find(self, query="", category="", provides="", requires="", fields=[]):
```

Comment:
`provides` and `requires` should be a Python List.
```suggestion
    def find(self, query="", category="", provides=[], requires=[], fields=[]):
```

And then, you should add two conditionals like the one we have for `fields`.